src/app/clarin-licenses/clarin-all-licenses-page/clarin-all-licenses-page.component.scss

```diff
@@ -0,0 +1,22 @@
+.label-PUB {
+  background-color: #5cb811 !important;
```

Comment:
Try to remove !important. Remove as many of them as possible. According to my limited knowledge, this is very bad practice.

Reply:
There must be `!important` because there are overridden styles from `_clarin-styles.scss` in this `.scss` file.